docs/api.md

```diff
@@ -73,6 +74,21 @@ In case of error:
 }
 ```
 
+For the election related responses, the `Status` field is indicating whether the transaction including the request was included in the blockchain or not. If the transaction was not included, the `Status` field is set to `0`. Otherwise, it is set to `1`.
```

Comment:
The wording seems off, isn't it transaction included in the request?
```suggestion
For the election related responses, the `Status` field is indicating whether the transaction included in the request was included in the blockchain or not. If the transaction was not included, the `Status` field is set to `0`. Otherwise, it is set to `1`.
```